1.1/spec/20-Annex-D.adoc

```diff
@@ -0,0 +1,260 @@
+= Annex E - Alignments (informative)
+
+== E.0 Overview
+
+This Annex provides alignments of GeoSPARQL to other well known ontologies that are either commonly used with GeoSPARQL or could be.
+
+The prefixes used for the ontologies mapped to in all following sections are given in the following table.
+
+[frame=none, grid=none, cols="1, 6"]
+|===
+| dcterms: | http://purl.org/dc/terms/
+| geo: | http://www.opengis.net/ont/geosparql#
+| geom: | http://geovocab.org/geometry#
+| gn: | https://www.geonames.org/ontology#
+| juso: | http://rdfs.co/juso/
+| locn: | https://www.w3.org/ns/locn
+| pos: | http://www.w3.org/2003/01/geo/wgs84_pos#
+| prov: | http://www.w3.org/ns/prov#
+| rdf: | http://www.w3.org/1999/02/22-rdf-syntax-ns# 
+| rdfs: | http://www.w3.org/2000/01/rdf-schema#
+| sdo: | https://schema.org
+| sosa: | http://www.w3.org/ns/sosa/
+| spatial: | http://geovocab.org/spatial#
+| ssn: | http://www.w3.org/ns/ssn/
+| time: | http://www.w3.org/2006/time#
+|===
+
+== E.1 ISA Programme Location Core Vocabulary (LOCN)
+
+LOCN Source: https://www.w3.org/ns/locn
+
+The LOCN specification provides notes on the use of GeoSPARQL literals (see https://www.w3.org/ns/locn#changes).
+
+|===
+| From Element | Mapping relation | To Element | Notes
+
+| `geo:Feature` | `rdfs:subClassOf` | `dcterms:Location` | LOCN states that `dcterms:Location` "represents any location, irrespective of size or other restriction", thus it is a superclass of `geo:Feature` 
+| `locn:Address` | `rdfs:subClassOf` | `geo:Feature` | Although LOCN indicates no spatial or geometry properties for `locn:Address`, `locn:Address` is clearly some specailized form of a Feature
+| `geo:Geometry` | `rdfs:subClassOf` | `locn:Geometry` | The LOCN class "defines the notion of "geometry" at the conceptual level, and it shall be encoded by using different formats", so GeoSPARQL's class is more specialised.
+| `geo:hasGeometry` | `rdfs:subPropertyOf` | `locn:geometry` | The LOCN property notes say "Depending on how a geometry is encoded, the range of this property may be one of..." so GeoSPARQL's proerty is learly more specialized.
+|===
+
+== E.2 WGS84 Geo Positioning: an RDF vocabulary (POS)
+
+POS Source: http://www.w3.org/2003/01/geo/
+
+|===
+| From Element | Mapping relation | To Element | Notes
+
+| `geo:SpatialObject` | `owl:equivalentClass` | `pos:SpatialThing` | Both classes are unrestricted, essentially abstract classes
+| `pos:Point` | `rdfs:subClassOf` | `geo:Geometry` | Via `pos:Point rdfs:subClassOf pos:SpatialThing` but since `pos:Point` usage notes indicate sirect postitioning, it is a form of geometry
+| `pos:Point` | `rdfs:subClassOf` | `sf:Point` | 
```

Comment:
Is `pos:Point` really more specialized than `sf:Point` ?

Reply:
No, adopted equivalence